gnmi/gnmi.go

```diff
@@ -200,12 +200,14 @@ func updateCache(cache *cache.Cache, dirtyRoot, root ygot.GoStruct, target, orig
 			return fmt.Errorf("gnmi: %v", err)
 		}
 	} else {
-		n, err := ygot.Diff(root, dirtyRoot, &ygot.DiffPathOpt{PreferShadowPath: preferShadowPath})
+		ns, err := ygot.DiffWithAtomic(root, dirtyRoot, &ygot.DiffPathOpt{PreferShadowPath: preferShadowPath})
```

Comment:
no action required: is this gonna to affect gNMI performance?

Reply:
Added benchmark. performance was not negatively affected.